<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Demo Suite" duration-ms="22109" started-at="2016-03-09T10:32:17Z" finished-at="2016-03-09T10:32:39Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="22109" started-at="2016-03-09T10:32:17Z" finished-at="2016-03-09T10:32:39Z">
      <class name="com.sample.SampleTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.sample.SampleTest@66d33a]" name="setUp" is-config="true" duration-ms="14437" started-at="2016-03-09T16:02:17Z" finished-at="2016-03-09T16:02:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testTitle()[pri:0, instance:com.sample.SampleTest@66d33a]" name="testTitle" duration-ms="2547" started-at="2016-03-09T16:02:32Z" finished-at="2016-03-09T16:02:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The Title of the FirstPage is not correct expected [Google] but found [goog]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The Title of the FirstPage is not correct expected [Google] but found [goog]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.sample.SampleTest.testTitle(SampleTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTitle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.sample.SampleTest@66d33a]" name="tearDown" is-config="true" duration-ms="5062" started-at="2016-03-09T16:02:34Z" finished-at="2016-03-09T16:02:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.sample.SampleTest -->
    </test> <!-- Test -->
  </suite> <!-- Demo Suite -->
</testng-results>
